frontend/src/main/pages/ChatHistoryPage.jsx

```diff
@@ -0,0 +1,178 @@
+import React, { useEffect, useRef } from "react";
+import { Button, Spinner } from "react-bootstrap";
+import { useParams, useNavigate } from "react-router";
+import { useInfiniteQuery } from "react-query";
+import axios from "axios";
+
+import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import ChatMessageDisplay from "main/components/Chat/ChatMessageDisplay";
+import ChatMessageCreate from "main/components/Chat/ChatMessageCreate";
+import { useBackend } from "main/utils/useBackend";
+
+const PAGE_SIZE = 25;
+const REFRESH_RATE = 2000;
+
+const ChatHistoryPage = () => {
+  const { commonsId } = useParams();
+  const navigate = useNavigate();
+  const loadMoreRef = useRef(null);
+
+  // Stryker disable all
```

Comment:
New code is not permitted to use `// Stryker disable all` for this kind of code; we now know how to mock useBacked calls.

Please see: https://ucsb-cs156.github.io/topics/stryker/stryker_useBackend_stryker_exceptions.html